pkg/catalogv2/oci/client.go

```diff
@@ -228,13 +228,31 @@ func (o *Client) getAuthClient() (*http.Client, error) {
 		if o.exponentialBackOffValues.MaxRetries > 0 {
 			retryPolicy.MaxRetry = o.exponentialBackOffValues.MaxRetries
 		}
-		if o.exponentialBackOffValues.MaxWait != nil {
-			retryPolicy.MaxWait = o.exponentialBackOffValues.MaxWait.Duration
+		if o.exponentialBackOffValues.MaxWait != "" {
+			maxWait, err := time.ParseDuration(o.exponentialBackOffValues.MaxWait)
+			if err != nil {
+				return nil, err
```

Comment:
For improved troubleshooting:
```suggestion
				return nil, fmt.Errorf("failed to parse exponentialBackOffValues.MaxWait: %w", err)
```

Reply:
Done.